int32[] obst_index
geometry_msgs/PoseStamped obst_pose
geometry_msgs/Twist obst_vel